Scripts/GradientsTypeScript.swift

```diff
@@ -2,45 +2,69 @@
 
 import Foundation
 
+func getColorLiteral(hexString: String)->String{
+  let hex = hexString.trimmingCharacters(in: CharacterSet.alphanumerics.inverted)
+  var int = UInt32()
+  Scanner(string: hex).scanHexInt32(&int)
+  let a, r, g, b: UInt32
+  switch hex.characters.count {
+  case 3:
+    (a, r, g, b) = (255, (int >> 8) * 17, (int >> 4 & 0xF) * 17, (int & 0xF) * 17)
+  case 6:
+    (a, r, g, b) = (255, int >> 16, int >> 8 & 0xFF, int & 0xFF)
+  case 8:
+    (a, r, g, b) = (int >> 24, int >> 16 & 0xFF, int >> 8 & 0xFF, int & 0xFF)
+  default:
+    (a, r, g, b) = (1, 1, 1, 0)
+  }
+  
+  return "#colorLiteral(red: \(CGFloat(r) / 255), green: \(CGFloat(g) / 255), blue: \(CGFloat(b) / 255), alpha: \(CGFloat(a) / 255))"
+}
+
+
+//MARK: - Script
+
 let gradientTypeURL = "https://raw.githubusercontent.com/Ghosh/uiGradients/master/gradients.json"
 
-func JSONFromURL(urlToRequest: String) -> NSData {
-    return NSData(contentsOfURL: NSURL(string: urlToRequest)!)!
+func JSON(_ urlToRequest: String) -> Data {
+  return (try! Data(contentsOf: URL(string: urlToRequest)!))
```

Comment:
Instead of force unwrap, won't it be better to use a `guard ... else { fatalError(...) }`? If the script fails, it that fails, it would be for the best that the script stop running